lib/annotations/example.js

```diff
@@ -61,28 +61,43 @@ module.exports = (env) => {
             exampleItem.rendered = undefined;
             /* istanbul ignore else */
             if (!sass) {
+              if (
```

Comment:
I'm wondering of this new code might make sense factored out as a util instead of inline?

Reply:
Moved in https://github.com/oddbird/sassdoc-theme-herman/pull/416/commits/f30775eaffcfa225aa51781ff93e81c45ea548ad